frontend/src/pages/destructionlist/detail/constants.ts

```diff
@@ -2,7 +2,10 @@ import { BadgeProps } from "@maykin-ui/admin-ui";
 
 import { DestructionListStatus } from "../../../lib/api/destructionLists";
 
-export const STATUSES_ELIGIBLE_FOR_EDIT = ["changes_requested"];
+export const STATUSES_ELIGIBLE_FOR_EDIT = [
+  "changes_requested",
+  "internally_reviewed", // TODO: Double check if we should make a entirely new check, or if this is correct
```

Comment:
I don't think we should be able to edit the destruction list after approval? However we do need to be able to visit the detail page, but without any editing options except assigning the archivist. This status implies that it's editable so I think a new check should be added. Also: the detail page should reflect this check.